Makefile

```diff
@@ -29,6 +29,9 @@ generate: manifests **.md
 manifests: examples/kustomize.jsonnet $(GOJSONTOYAML_BIN) vendor build.sh
 	./build.sh $<
 
+update_libs: $(JB_BIN)
+	$(JB_BIN) update
+
```

Comment:
I don't think we need this additional target. It's not use as dependency by another one and only adds one single command.